conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureHandler.java

```diff
@@ -88,11 +94,27 @@ public ConjureHandler delete(String template, HttpHandler handler) {
 
     private ConjureHandler register(HttpString method, String template, HttpHandler handler) {
         HttpHandler current = handler;
-        String endpoint = method + " " + template;
-        for (BiFunction<String, HttpHandler, HttpHandler> wrapper : WRAPPERS) {
+        EndpointDetails endpoint = new EndpointDetails(method, template);
+        for (BiFunction<EndpointDetails, HttpHandler, HttpHandler> wrapper : WRAPPERS) {
             current = wrapper.apply(endpoint, current);
         }
         routingHandler.add(method, template, current);
         return this;
     }
+
+    private static final class EndpointDetails {
```

Comment:
Seems like the only thing you can do with an EndpointDetails is to display it. That seems like a weird abstraction. Either add getters (or make it immutable) or remove the class in favor of a static "String displayEndpoint".

Reply:
We use it for the cache-control handler to avoid handler registration for non-GET requests. @uschi2000 requested that I remove the getters in an earlier review.